### ISSUE TITLE
Aria label missing for close button

### ISSUE THREAD

**Issue by Rafi993** [forem/forem#11859]
**Describe the bug**

Close button in the banner has icon but does not have corresponding text associated with it so screen reader user might not know what the button does clearly. Adding aria label will help

**To Reproduce**

1. Go to https://dev.to/
2. Make sure it campaign banner at the top
3. Now try navigating the page with screen reader to see what the close button in the banner say

**Expected behavior**

When focused with screen reader on the close button should describe what it does

**Screenshots**

![closeButton](https://user-images.githubusercontent.com/8092120/101854290-3faf2a00-3b87-11eb-8896-126da74eeadc.jpg)

**Desktop:**

- OS, version: Ubuntu 20.04
- Browser, version: Version 1.15.75 Chromium: 86.0.4240.99 (Official Build) (64-bit)

**Comment by Rafi993** [comment]
The "X" icon is actually represented using character "x" but adding aria-label="close button" might make it more explicit.

**Comment by andreamartz** [comment]
I'd like to work on this.

**Comment by nickytonline** [comment]
Assigning this to you @andreamartz.
